### DISCUSSION TITLE
Cant Flash DWIN_SET (Gotcha) UI Update

### DISCUSSION THREAD

**Discussion by thesamgordon** [Jyers/Marlin#1768] [Q&A] [answered]
I just installed Jyers 2.0 (It's amazing BTW) and I wanted to flash the icon pack called Gotcha. I followed all the instructions. I am using the stock SD card for my Ender 3 V2, I have it formatted as a FAT32 with 4k allocations, I have renamed the file to DWIN_SET, and put it into the SD slot on the back of the motherboard on the screen. When I boot up with the card, it's locked on the Creality logo until I press any button. After pressing a button and being brought to the home screen, I see "SD Init Fail" at the bottom.  Im at a loss for ideas.

**Comment by Cal-LF** [comment]
The SD card from Creality should not be used. Try another SD card.
See #1311 #1401

**Reply by thesamgordon** [reply]
Using another SD card does not allow for 4k allocations. I can do 4, 16, 32, 64, or 8k. Does it need to be 4k?

**Reply by winrun44** [reply]
There are two firmwares corresponding to two different screens. For DWIN screens you need a firmware with a folder [DWIN_SET] and for DACAI screens you need a firmware with a folder [private].

**Comment by Cal-LF** [comment]
Yes. If using Windows 4096 is supported.

**Reply by thesamgordon** [reply]
It's not an option in the windows formatting tool.

**Comment by Cal-LF** [comment]
![format](https://user-images.githubusercontent.com/76499230/148709608-21f6d62a-f15d-4ee6-b62e-095e75bbd8fd.jpg)

**Reply by thesamgordon** [reply]
I'm using a 32 gig card. Does that not support 4k allocations?

**Reply by Cal-LF** [reply]
@Loudbooks Partition to 8g or 16g.

**Reply by thesamgordon** [reply]
Okay I can try that

**Reply by thesamgordon** [reply]
@Cal-LF That did not work. I partitioned a different SD card to 8 GB, made it FAT32 with 4k allocations, and I still get SD Init Fail

**Comment by method1020** [comment]
I think you need to use 8GB or less.

**Reply by thesamgordon** [reply]
:/ is there any other way?

**Reply by method1020** [reply]
I have used the card that came with the printer in the past. It worked fine when formatted correctly.

**Reply by blackoutlaw** [reply]
are you putting it in the printer or the display sd slot it needs to go into the display sd slot

**Reply by thesamgordon** [reply]
I am putting it in the display SD slot.

**Comment by whitefalcon684** [comment]
@Loudbooks , 

Did you ever find a solution to your issue? I have been having the EXACT same issue and have been feeling like I was crazy because It didn't seem like anyone else was having the same issue. Either they were getting the blue screen and going to orange immediately or it worked.

**Reply by Holypug** [reply]
I have the same. Tryed with many sd cards of various size . Display won't update.

**Reply by 3dNoobie619** [reply]
same her i have tried different sd cards i have tried the DWIN_SET] and for DACAI private and all i get is a black screen or a boot screen sayig there is no file at this point just going back to factory would even be better then nothing

**Comment by SiliconExarch** [comment, marked as answer]
Out of interest does your screen PCB say "Rev 1.20" anywhere on it? I'm also unable to update the firmware with the same symptoms as is someone on Reddit and we both have this in common. The fact we're not seeing the blue/orange screens at all makes me think it's not finding the firmware, but it is clearly recognising the micro SD because it doesn't boot "normally".

**EDIT:** My Ender 3 V2 was purchased very recently but came with a 4.2.2 mainboard.

**Reply by jameskirk** [reply]
Where would I find out the Rev number at can you take a snapshot please?
Thank you!

**Reply by whitefalcon684** [reply]
My Screen PCB does say "Rev 1.2". 

My Ender 3 v2 was also purchased 2 weeks ago and came with the 4.2.2 Mainboard. 


@jameskirk - Here is a picture of my Screen PCB you can see in the bottom right hand side it says Rev 1.2 
![20220130_101233](https://user-images.githubusercontent.com/3135828/151705497-310619e7-46d6-4694-a61b-99181f022fe6.jpg)

**EDIT:** @SiliconExarch  can you link the reddit thread?

**Reply by SiliconExarch** [reply]
Please see below:

![DSC_0894](https://user-images.githubusercontent.com/25375403/151705681-61f2d6da-40e5-4b69-9c23-7dc67036b5ed.JPG)

This is definitely a totally different LCD module compared to the "old" Ender V2 screen, which used a DMT48270C043_04WN. I've had no luck searching for either the part number (3SZCX48270M043) or what I assume is the board ID (3SZCX01A) but it uses the same main IC as [this module I found](http://www.gz-dc.com/article/id/6816.html) so I'm going to see if I can dig up any information on how to program it.

**Reply by jameskirk** [reply]
Here is a picture of mine I don't see a Rev number on it.
![20220130_132223](https://user-images.githubusercontent.com/91149294/151712335-e139f5d4-4a4e-4b66-989d-0225601970ff.jpg)

**Reply by thesamgordon** [reply]
This is going a lot deeper than I thought it would. Is there anything I can do to help?

**Reply by SiliconExarch** [reply]
> Here is a picture of mine I don't see a Rev number on it. ![20220130_132223](https://user-images.githubusercontent.com/91149294/151712335-e139f5d4-4a4e-4b66-989d-0225601970ff.jpg)

Unfortunately that looks like the standard screen, so either the micro SD slot is faulty (I wonder if that's what the mark on it is supposed to signify?) or you're not preparing the card correctly, but more likely the former since you seem to know what you're doing.

**Reply by SiliconExarch** [reply]
> This is going a lot deeper than I thought it would. Is there anything I can do to help?

Not much, I think I'm making good progress. I'll share my files as soon as I've confirmed they're safe to flash.

![DSC_0897](https://user-images.githubusercontent.com/25375403/151717606-9835dafc-d958-417c-b113-2ec2bb9512c5.JPG)
![DSC_0903](https://user-images.githubusercontent.com/25375403/151717650-4841b35d-45cd-4fd1-97f7-80d655354467.JPG)

**Reply by thesamgordon** [reply]
Super coder human saves that day

**Reply by SiliconExarch** [reply]
I've fixed the bug with the solid background colour behind the text and converted all 3 variations:
[New display firmware.zip](https://github.com/Jyers/Marlin/files/7967323/New.display.firmware.zip)

Copy the folder for your desired flavour to the root of your micro SD and rename it to private then slot it into the LCD and switch on the printer, you should see the blue screen above with text & a progress bar. When it's complete and the screen changes, power cycle the printer once more with the micro SD removed and you should be good to go.

Please note that this will replace the stock font with NK57 Monospace Condensed. If you'd rather use a different font (must be TTF format), rename it to font.ttf and copy it inside the truefont folder, replacing the existing file then flash the screen again. If for some reason you really liked the stock font, you can find it in the xgus conversion tool download from the [manufacturer's website](http://www.gz-dc.com/category/typeid/412#mainTop).

**Reply by K2Kevin** [reply]
Nice work, I would like to ONLY change the font.  I remember trying months back and it would never apply just that font.  Do you know how to do that?  I think I just removed all other files and left the font file (can't remember what I did, but it failed).

**Reply by SiliconExarch** [reply]
> Nice work, I would like to ONLY change the font. I remember trying months back and it would never apply just that font. Do you know how to do that? I think I just removed all other files and left the font file (can't remember what I did, but it failed).

If we're talking about the new screens, you'd have to flash the entire contents of the private (Original) folder with font.ttf swapped out for the one you want or it'll wipe the config.txt and any images which were already loaded onto the screen. I'm going to attempt to contact Creality for a copy of the actual stock firmware but in the meantime that's the best you can do.

Make another thread if you want to discuss the old DWIN screens since they're a whole different kettle of fish with their lack of support for TrueType fonts.

**Reply by K2Kevin** [reply]
Actually, the stock screens are included in Jyers download, and/or elsewhere.  It's available.

**Reply by SiliconExarch** [reply]
> Actually, the stock screens are included in Jyers download, and/or elsewhere. It's available.

I'm pretty sure the stock screens for this new LCD module that's only been spotted in the wild for less than a month aren't available to download anywhere but I've spent enough time on this today so I'm out...

**Reply by whitefalcon684** [reply]
@SiliconExarch 

Thanks for all the hard work getting this figured out! Going to try out flashing the new screen firmware this morning. Will post back with the results on my end. 

Edit: 

We have success! Thanks again @SiliconExarch  
![20220131_091226](https://user-images.githubusercontent.com/3135828/151809777-7c409a22-3857-4fe5-8af0-6ea4b44d544e.jpg)
![20220131_091605](https://user-images.githubusercontent.com/3135828/151809790-8858af22-0114-4acc-88cd-aaa42b048caf.jpg)

**Reply by lostboy4480** [reply]
Thank you so much for this,  I got my printer the other day, Mainboard v4.2.2 and couldn't get the screen flash to work.  Opened it up and it is a 1.2 revision board.  Tried your zip and it flashed straight away.

Thank you !

**Reply by m9365428** [reply]
@Loudbooks looks like a fix to me. Please mark the solution.
@SiliconExarch You are the man. Looks like the new units are coming with Rev 1.2 displays. PLEASE make a PULL REQUEST with your data, files, and method.
I tried it with a Onn 32GB partitioned to 4GB, 4096 bytes Allocation unit, with both private and DWIN_SET on the card. Unit loaded successfully. Only issue was after the reboot the screen gets stuck in a not populated mode but after a reboot with out the card inserted it works as normal.
May be possible to have both folders in a Gothic folder and change instructions to contents of folder so users don't have to identify the display version.

**Reply by thesamgordon** [reply]
Alright!

**Reply by Holypug** [reply]
My display still don't want be updated. Damn

**Reply by marcribe** [reply]
> @SiliconExarch Hey Vanessa, it would be cool of you to share your method or your scripts to create and read .zico files...

this will be nice... I have a lot of custom icons (translated to portuguese) that I want to convert to this new format

**Reply by PatrickKinz** [reply]
Got my printer today and it has a Rev. 1.3 board. This solution from @SiliconExarch still worked. Super cool

**Reply by tico74** [reply]
any chance someone could make a video of this for us that are inexperienced? is there a way to revert back stock firmware on LCD after this update for dacai LCD screens?

**Reply by Rick507** [reply]
Thank you so much for this. My Screen PCB does say "Rev 1.4" and rescued.

![IMG_4238](https://user-images.githubusercontent.com/38333671/161450094-ee2f58e4-6e49-4d98-b930-99987efc66d5.JPG)

**Reply by I-wish-2** [reply]
Greetings All  just assembled my ender3 V2 today and and flashed the M.B 4.2.2 and that worked great.  The display not so much.  I worked my way to this site and found " SiliconExarch's" fix, THANK YOU ever so much.  I have the Rev 1.3 display board just to add one more rev to the pile.  Thanks again to all the comments and for this great site.  OOOPS back to 3D class.

Endeavor to Persevere

**Reply by samuelT2** [reply]
@SiliconExarch, you just saved me a night shift, thanks a thousand! After a lot of troubles getting jyers with ubl and bltouch to run on my new Ender 3 V2, the icons upgrade have almost driven me crazy, until I found this [reddit](https://www.reddit.com/r/ender3v2/comments/stl43i/ender_3_v2_lcd_firmware_dwin_set_wont_flash/) and finally this thread. I'd tip you a beer for sure! I'd probably never have spotted the difference within the boards by myself...

**Reply by prariegold** [reply]
> I've fixed the bug with the solid background colour behind the text and converted all 3 variations: [New display firmware.zip](https://github.com/Jyers/Marlin/files/7967323/New.display.firmware.zip)
> 
> Copy the folder for your desired flavour to the root of your micro SD and rename it to private then slot it into the LCD and switch on the printer, you should see the blue screen above with text & a progress bar. When it's complete and the screen changes, power cycle the printer once more with the micro SD removed and you should be good to go.
> 
> Please note that this will replace the stock font with NK57 Monospace Condensed. If you'd rather use a different font (must be TTF format), rename it to font.ttf and copy it inside the truefont folder, replacing the existing file then flash the screen again. If for some reason you really liked the stock font, you can find it in the xgus conversion tool download from the [manufacturer's website](http://www.gz-dc.com/category/typeid/412#mainTop).

After 3 days of frustration I found this thread!  I can’t thank you enough for this fix!! Nicely done 👏🏻👏🏻

**Reply by Onejk1** [reply]
Not sure this is closed as the answer as look at the status screen. V1.4 board with Gotcha icon set
![5228C003-FBEA-4481-B4DF-6A1C415F3A9C](https://user-images.githubusercontent.com/30779695/170113191-4b55821f-a6f7-482d-93d1-61ad4ca22da7.jpeg)

**Reply by maiseld** [reply]
> I've fixed the bug with the solid background colour behind the text and converted all 3 variations: [New display firmware.zip](https://github.com/Jyers/Marlin/files/7967323/New.display.firmware.zip)
> 
> Copy the folder for your desired flavour to the root of your micro SD and rename it to private then slot it into the LCD and switch on the printer, you should see the blue screen above with text & a progress bar. When it's complete and the screen changes, power cycle the printer once more with the micro SD removed and you should be good to go.
> 
> Please note that this will replace the stock font with NK57 Monospace Condensed. If you'd rather use a different font (must be TTF format), rename it to font.ttf and copy it inside the truefont folder, replacing the existing file then flash the screen again. If for some reason you really liked the stock font, you can find it in the xgus conversion tool download from the [manufacturer's website](http://www.gz-dc.com/category/typeid/412#mainTop).

I have a Rev 1.2 LCD board and after flashing I see only images and no text (unless I look into file folders). Can you please fix this?

**Reply by lexlix** [reply]
I got the Rev 1.4 Display and 4.2.2 Board. Your Display firmware saved my screen from showing just a black screen after flashing the default firmware from Creality @SiliconExarch . But now I am stuck at the boot screen just showing the Creality Image. I tried to flash every Mainboard firmware I could find but no luck. any ideas
![IMG_2546](https://user-images.githubusercontent.com/6112217/183247885-dd3babc7-6f8b-4ef8-8417-b88979c0edb7.jpg)
?

**Reply by lexlix** [reply]
Also does anyone maybe have to default Creality firmware for the display Rev1.4? That might be a good start to get my printer running again

**Reply by AkARIO1224** [reply]
> I've fixed the bug with the solid background colour behind the text and converted all 3 variations: [New display firmware.zip](https://github.com/Jyers/Marlin/files/7967323/New.display.firmware.zip)
> 
> Copy the folder for your desired flavour to the root of your micro SD and rename it to private then slot it into the LCD and switch on the printer, you should see the blue screen above with text & a progress bar. When it's complete and the screen changes, power cycle the printer once more with the micro SD removed and you should be good to go.
> 
> Please note that this will replace the stock font with NK57 Monospace Condensed. If you'd rather use a different font (must be TTF format), rename it to font.ttf and copy it inside the truefont folder, replacing the existing file then flash the screen again. If for some reason you really liked the stock font, you can find it in the xgus conversion tool download from the [manufacturer's website](http://www.gz-dc.com/category/typeid/412#mainTop).

Hey so Ive been trying to update my firmware with my ender 3v2. Im able to get to the blue screen and it updates properly but when I try to flash the new jyver UI its stuck at the creality logo. Any idea on why this is?

**Reply by kmart2000** [reply]
i tried doing that now my screen just stays black

**Reply by jonwhick** [reply]
I am having this problem with a Rev 1.5 display, AND a BTT SKR e3 V3. It just gets stuck on this screen. Haven't tried flashing the display, but I currently am unable to flash any firmware to the board.

**Reply by murthara** [reply]
Has anyone fixed the above issue with updating the display? I'm still stuck at the Creality logo.

**Reply by Killj0y1** [reply]
Can confirm this worked when many other things didn't. I could get it to see firmware and flash it it's a dos looking screen but then black screen after even tried flashing the main board after to see if it was just no communication. Used a 2gig card and the included one. Did formats many times as per recommendations. Even tried a version of crealitys which I found was too big because it was for another model (which now I wonder if we can when get the stock firmware in case things go south I didn't find it on their official site). Well this file worked then I flashed the main board and voila looks good. 

I have a rev 1.2. seems there was 3 revisions I was l found a visual aid but can't find it again. To post. Either way this worked so worth a try if you're at your wits end. I would love to know what exactly is different with this version so we can make future versions (assuming there is any) work.

**Reply by 3dNoobie619** [reply]
Hi im new to all this and from what i can tell im having a very similar experience as this person 
I have an 
Ender 3 V2
4.2.2 mother board
gigabyte card in the u7 spot
I upgraded to the sprite pro extruder/hotend and was trying to flash the firmware to be able to use it and could not gete it to work. i contacted creality and they gave me firmware that was supposed to be correct and told me to rename the file firmware.bin  that finally got it to boot but now all i have is a black screen.The screenwill not boot and folders it will only boot if the file is firmware.bin please help if you can

**Reply by 3dNoobie619** [reply]
![20230622_024858](https://github.com/Jyers/Marlin/assets/137449946/c9963388-60f8-4735-b2fe-ccd3162dbb10)

**Reply by 3dNoobie619** [reply]
i feel like i need to start from the beginning and redo everything so even a factory reset would be fine

**Reply by 3dNoobie619** [reply]
the screen i get if the file is not named firmwear.bin
![Uploading 20230621_223008.jpg…]()

**Reply by 3dNoobie619** [reply]
also i am new to this page and everything seems all over the place so if i did get an answer then i dont know how to tell lol

**Reply by RV-from-be** [reply]
@3dNoobie619 First, Jyers has abandoned all development for a year.  Secondly, your screen is neither a DWIN, nor a DACAI, nor a TJC but a SYMWIT.  TJC and SYMWIT cannot have custom icons.  So goodbye Gotcha packs or others.  Additionally all font sizes used in v2.0.1 of Jyers are only usable.  The SYMWIT is the worst screen provided by Creality.  It was quickly abandoned for first the DACAI then now by the TJC.  Give me your CPU type and your motherboard version and I will provide you with a firmware that can be compatible on a very advanced basis of the current version of Jyers.

**Reply by UrsidaeMajor** [reply]
I've tried the above suggestions and nothing has worked! Any help is appreciated.

**Comment by SommerYou** [comment]
![20220502_222121](https://user-images.githubusercontent.com/70163113/166395978-72e30f22-a45a-4d55-8b9b-99a9d2679e09.jpg)
![20220502_222136](https://user-images.githubusercontent.com/70163113/166395980-cb17372d-48ce-400b-877a-559d84ee4689.jpg)
I had all the problems mentioned here...could not get the screen to update.  I used SiliconExarch's zip file.  It updated so it was good to know the LCD board was no problematic but after trying all private updates, my screen is garbled.  There are no images, just text.  It says Creality 26 26 (no icons) on the main screen.  On some screens no text even, just some lines.

**Reply by Malice359** [reply]
I'm the same...... kind of. I have the images but no text I have Rev 1.4.

**Reply by peterudo** [reply]
I'm having this issue too. Did you find a solution?

**Reply by peterudo** [reply]
I was trying the MarlinUI.

**Reply by peterudo** [reply]
I tried the CrealityUI now, which looked even more messed up. It flashes and updates, but no icons are showing.

**Reply by Onejk1** [reply]
![19CCFFAB-0EFC-4A4C-8C33-565BC3FE9F54](https://user-images.githubusercontent.com/30779695/170112873-ceb3ff98-5184-4e41-906d-456bb2f3db95.jpeg)
I’m not sure this is fully sorted. I have a 1.4 board that I flashed as per the new instructions. Look at the status screen, issues with elapsed time and random characters by the file name
![Uploading B9A18710-F86B-4EAF-8788-72CE65341693.jpeg…]()

**Reply by muffinb0x** [reply]
Heyo! So I had the same black screen problem, and this was a huge fix! Except.... the only language on the LCD is Chinese now. If I try to change the language, the display either goes back to the Creality screen with the number "21" on it in small white font, or there just aren't any characters at all under any of the various menu options. Anyone else run into this, or have any tips?

Edit: I have a Rev 1.3 board.

Edit 2: I got this fixed! I flashed my mobo back to the mriscoc [Ender3V2-422-MM-20220527.bin](https://github.com/mriscoc/Ender3V2S1/releases/download/20220527/Ender3V2-422-MM-20220527.bin), and everything works as it should :)

**Comment by Realdutchi** [comment]
Is it save to use this screen firmware on the Rev 1.4 board or is that still in development?

**Comment by EnzohTV** [comment]
I have the ender 3 v2 with the 4.2.2 mainboard and rev 1.4 screen and my printer has been running fine for a few weeks now with the firmware. I used the 5x5 auto level as i have the cr touch, i also couldnt get it to install the screen firmware but eventually managed to do it. I used the file named "New.Display.Firmware" and inserted the sd card with 2 identical "private (Gotcha)", i called 1 "private" and the other "DWIN_SET". This then ran what you would typically identify as code and once it was complete i turned it off and removed the sd card then started it back up again with the card empty (then removed it after i knew it worked) and the printer has worked fine since with the new screen firmware. I have attached the file "New.Display.Firmware". If this doesnt work i may have missed a step that i did but i cannot remember so please reply and i should see it as soon as possible. Sorry if this did not work for you. (Do this at your own risk as with any printer mod).
[New.display.firmware.zip](https://github.com/Jyers/Marlin/files/9275515/New.display.firmware.zip)
.
After seeing peoples comments please remember to do all steps necessassary to install jyers firmware prior to trying other methods to fix issues.

**Reply by jdvann** [reply]
I used this update screen rev 1.5 it worked fine

**Reply by nicolasaigner** [reply]
Thanks so much for sharing your knowledge! It worked here! I did exactly what you did. I had trying to put the firmware "E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin" without updating the display board.
My screen was stuck with the Creality logo.
Following the step by step you reported worked:
1 - I unzipped the "New.display.firmware";
2 - I copied the "private (Gotcha)" into the SD;
3 - I renamed the folder to "private";
4 - I made a copy of the folder;
5 - I renamed the copied folder to "DWIN_SET";
6 - With the printer turned off, I placed the SD card on the display board;
7 - I turned on the printer and started to update the files, at the end a message appears that the update ended;
8 - I turned off the printer, took out the SD card;
9 - I deleted everything from the card;
10 - I put the firmware file "E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin" on the card;
11 - I put the SD card in the printer;
12 - I turned on the printer and in less than 10 seconds it updated;

Thank you very much!
Printer: Ender 3 v2
Board version: 4.2.2
Display board version: 1.4

**Reply by ValokAsura** [reply]
My thanks to you as well!

It took several tries for the 4.2.2 board firmware to update. After the first attempt, the screen would always go straight to the "Creality" loading screen and remain indefinitely until I powered off the machine. I thought I killed it. On the 5th try or so, after formatting the SD card a couple of times, renaming the firmware each time, and eventually settling on pleasework.bin, I came back to the printer and it had finally loaded up with the new firmware.

I couldn't flash the screen from the Marlin-2.0.1.zip images, but downloading and using your New.display.firmware.zip worked a treat!

Printer: Ender 3 v2
Board version: 4.2.2
Display board version: 1.5

**Reply by AkARIO1224** [reply]
> Thanks so much for sharing your knowledge! It worked here! I did exactly what you did. I had trying to put the firmware "E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin" without updating the display board. My screen was stuck with the Creality logo. Following the step by step you reported worked: 1 - I unzipped the "New.display.firmware"; 2 - I copied the "private (Gotcha)" into the SD; 3 - I renamed the folder to "private"; 4 - I made a copy of the folder; 5 - I renamed the copied folder to "DWIN_SET"; 6 - With the printer turned off, I placed the SD card on the display board; 7 - I turned on the printer and started to update the files, at the end a message appears that the update ended; 8 - I turned off the printer, took out the SD card; 9 - I deleted everything from the card; 10 - I put the firmware file "E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin" on the card; 11 - I put the SD card in the printer; 12 - I turned on the printer and in less than 10 seconds it updated;
> 
> Thank you very much! Printer: Ender 3 v2 Board version: 4.2.2 Display board version: 1.4

Im able to flash the screen with private and DWINSET and it went through the update process. Im still not able to update with the bin file which I'm currently struggling on. Any tips?

**Reply by nicolasaigner** [reply]
@AkARIO1224 , sorry, I didn't see the message!
Still having problems? In your case, the bin file you speak of the firmware of the printer or the display?

Check your printer board and display model.
If it's the same as mine, follow this step by step that I answered there.

**Reply by DIVMSTR** [reply]
First off thank you!  I had a worn out selector button/knob and had to replace it.  For the life of me, I could not get my screen to update back to the Jyers that I had on there.  This worked for me.  I had been running Jyers already on the machine and the screen, so I did not have to update the machine, just the new screen.  I can confirm this works if running a 4.2.7 mother board as well.

Thank you very much!
Printer: Ender 3 v2
Board version: 4.2.7
Display board version: 1.4

Back to printing!
:-)

**Reply by vahmelk99** [reply]
Saved my day, thx a lot

**Comment by romanchoub** [comment]
Tried doing the update on my 1.4 screen and I'm only seeing the images but no text :(

**Reply by EnzohTV** [reply]
Sorry, i havent encountered that issue and have no knowledge on how that may be fixed, maybe try to reinstall the update or try different DWIN_SET files.

**Reply by romanchoub** [reply]
I guess I'll have to play around with a bunch of different firmwares. Hope I can find a fix, its my first 3d printer and still haven't gotten to enjoy it properly.

**Reply by EnzohTV** [reply]
Try to find some other pages about jyers and see if there is any more information for you to read and learn more about it because i doubt you will be the only one with this issue and i am quite new to 3d printing too so im sorry i cant help and if worst case scenario message creality and they will give you fresh firmware to reset your printer. I hope you can enjoy your printer soon <3.

**Reply by Realdutchi** [reply]
> I guess I'll have to play around with a bunch of different firmwares. Hope I can find a fix, its my first 3d printer and still haven't gotten to enjoy it properly. 

If you find a solution, please let us know?

**Reply by mannk32** [reply]
I just finished dealing with this issue. v4.2.2 main board, rev 1.4 display. I used the DWIN_SET and private files on a correctly formatted 8gb sd card and was getting no text issues on the display. You will need to update your main board to get the text to show up. I used jyers v2.0.1 specific to my v4.2.2 board and it worked right away

Also, I tried combinations of private and DWIN_SET before trying the main board update. I think just the DWIN_SET file is likely all you need

**Reply by Realdutchi** [reply]
Did you put the dwin and privat dirs on the card?

**Reply by mannk32** [reply]
First I used the sd card with DWIN and private to flash the display (inserting the sd into the back of the display), then I tested it to see if the new icons were showing up.

Then I cleared the sd card, copied the jyers .bin file onto it and inserted it into the front of the printer (where the micro-usb slot is) before turning it on. Worked immediately

**Reply by Realdutchi** [reply]
Thx, i ging to try that tomorrow!

**Reply by romanchoub** [reply]
Thank you all, followed Mannik32's steps and got it working!

**Reply by lexlix** [reply]
@mannk32 thank you for your research. I trining to follow your steps right now but Im unsure with firmware for the Display are you using? The New.display.firmware that is floating around here in the comments or the one in the jyers v2.0.1 source code.zip release?

**Reply by mannk32** [reply]
Sorry, editing to add: use the New.display.firmware files. Choose your preferred private file (I used Gotcha). Copy the private file onto your correctly formatted (please recheck your formatting first if you encounter issues) SD card. Load that file onto your display by pulling off the rear cover of the display and inserting the SD card into the only available slot and running your machine.

If the icons update and you are having text issues, move onto the mainboard update by emptying the SD card and downloading then copying the correct .bin file (specific to your main board, leveling system, preferred probe points and speed) on the Jyers v2.0.1 page to your SD card. Insert to your printers SD card slot, let it update, and you should have a fully functional updated display.

**Comment by Realdutchi** [comment]
Did you use the Privat version of the firmware?

**Reply by EnzohTV** [reply]
I used the normal one twice one called private and the other DWIN_SET but that just messed my screen up so i used the new display firmware for it after i did that, again one called DWIN_SET and private and this fixed the screen firmware for me as i had already installed the mainboard firmware.

**Comment by werkshopz** [comment]
I used a 16gb Scan Disk formated FAT32 w/ 4k Allocations and had no issues with flashing Icons.
Ender 3 v2 (4.2.2 Board)

**Comment by Hillbi11y** [comment]
Ok, same issue.  Read here and other places.  Everywhere says to put DWIN_SET on the SD card.... got to thinking.... I put private on the card too along with DWIN_SET and it worked like a champ!
Acted different, got blue screen but displayed many files being written scrolling down... said update complete and went to a funky creality screen... reboot, MARLIN (the fish) is there and all icons good :  )

**Comment by Myklt8** [comment]
I know this is a bit of an old thread, I get to this screen, it completes and then freezes on the Creality screen...buzzer screaming bloody murder the whole time.

I have an ender 3 with the 4.2.7 mb, and trying to install the V2 screen



![image](https://user-images.githubusercontent.com/126296783/221317513-d9be8065-3b95-4e14-88b8-b422d8dd5e5d.png)

**Comment by GAPManCA** [comment]
I have an Ender 3 v2 and the motherboard is actually v4.3.1.  In another thread I have learned that I should treat it like it was a v4.2.2 and updating the firmware of the printer work.  Also the upgrading of the screen worked perfectly for me using the New Display Firmware.zip file above.

**Comment by SheepDawgK** [comment]
Just put together an Ender 3 v2 bought straight from Creality's website.  v4.2.2 mainboard.  The screen's pcb version: 1.63 I believe as the QR code on the uSD card reader shows hxdb****xsdb43dc03v1.63+ and the silk screen shows TJC4827X243_011_P04.  I have installed the E3V2-BLTouch-5x5-HS-v4.2.2-v2.0.1.bin firmware to the printer just fine.  Then I downloaded the "New.display.firmware.zip", unzipped it, copied the "private (Gotcha)" folder to the formatted (with 4096B allocation unit size) 8GB uSD card the printer came with.  Removed the " (Gotcha)" from the folder name so it just says "private", copied the "private" folder locally on the card and renamed it "DWIN_SET".  When I put the uSD card in the Screen's uSD card reader, plug the ribbon cable back in and turn on the printer, it simply comes up with the original screen.  It does not turn solid blue, then orange when done, the icons are no different, it doesn't take 10 seconds of a blank screen, it's like it didn't read the uSD card at all.  When I turn it on, I see a progress bar that takes a couple of seconds and the same screen with original fonts and icons show up.  Any ideas of what I'm doing wrong? or what I could try?  Thanks in advance.
![E3V2_screen_pcb](https://user-images.githubusercontent.com/24484965/224502623-b0b23ef5-b
![E3V2_screen_main](https://user-images.githubusercontent.com/24484965/224502790-80f7b8c0-993b-4e8e-ab5c-5d28ef31e97e.jpg)
5a8-49fa-8607-66f94e0ff434.jpg)
![E3V2_screen_info](https://user-images.githubusercontent.com/24484965/224502803-36fc61ba-a425-4641-8d0e-cf886d453cd5.jpg)

**Reply by murray-m17** [reply]
I have the same screen as you and i'm having the same issue. did you ever find a fix? Thanks

**Comment by SheepDawgK** [comment]
Here is the PCB image.  It appears to be new, I can't tell if this is a DWIN or DACAI?
 ![E3V2_screen_pcb](https://user-images.githubusercontent.com/24484965/224520537-8f32520d-054f-4bcb-a378-610edfb4991f.jpg)

**Comment by winrun44** [comment]
It's TJC card.

**Reply by SheepDawgK** [reply]
Thanks!  I've searched the web and can't find anything about a TJC card.  Would you have any more info on it? or know if I would use the DWIN_SET or private or is this too new for support yet?

**Reply by winrun44** [reply]
It seems that this type of card poses problems https://github.com/mriscoc/Ender3V2S1/issues/542

**Reply by SheepDawgK** [reply]
Thanks, I guess I'll have to wait for support and just use the original graphics.

**Comment by 3dNoobie619** [comment]
idk if this will help me actually see wtf is going on on this site but im going to try to repost my issue here

Hi im new to all this and from what i can tell im having a very similar experience as people here
I have an
Ender 3 V2
4.2.2 mother board
gigabyte card in the u7 spot
I upgraded to the sprite pro extruder/hotend and was trying to flash the firmware to be able to use it and could not get it to work. i contacted creality and they gave me firmware that was supposed to be correct and told me to rename the file firmware.bin that finally got it to boot but now all i have is a black screen.The screenwill not boot folders it will only boot if the file is firmware.bin please help if you can

**Reply by 3dNoobie619** [reply]
at this point im ready to burn the @#$%ing thing so you might see me on youtube buring an ender 3

**Comment by 3dNoobie619** [comment]
Gigabyte and a 4.2.2

**Reply by samuelT2** [reply]
You should go over to https://github.com/mriscoc/Ender3V2S1 - his Marlin fork is up to date (with a very helpful community around).
